tests/foreman/cli/test_lifecycleenvironment.py

```diff
@@ -386,17 +240,16 @@ def test_positive_list_with_pagination(self):
         """
         # Test different `per-page` values
         for per_page in (1, 5, 20):
-            with self.subTest(per_page):
-                # Verify the first page contains exactly the same items count
-                # as `per-page` value
-                lces = LifecycleEnvironment.list(
-                    {'organization-id': self.org['id'], 'per-page': per_page}
-                )
-                self.assertEqual(len(lces), per_page)
-                # Verify pagination and total amount of pages by checking the
-                # items count on the last page
-                last_page = ceil(self.lces_count / per_page)
-                lces = LifecycleEnvironment.list(
-                    {'organization-id': self.org['id'], 'page': last_page, 'per-page': per_page}
-                )
-                self.assertEqual(len(lces), self.lces_count % per_page or per_page)
+            # Verify the first page contains exactly the same items count
```

Comment:
@latran you don't need to remove `with self.subTest(per_page):` if we are not parameterizing this test. Not a blocker though. Otherwise everything looks good to me

Reply:
alright, added back.